src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -216,6 +222,22 @@ class BinaryCompactObject : public DomainCreator<3> {
     static type default_value() { return true; }
   };
 
+  struct UseLogarithmicMapOuterSphericalShell {
+    using type = bool;
+    static constexpr OptionString help = {
+        "Use a logarithmically spaced radial grid in Layer 5, the outer "
+        "spherical shell."};
+    static type default_value() noexcept { return false; }
+  };
+
+  struct OuterLayerRadialRefinements {
```

Comment:
`Refinements` -> `Refinement` or `RefinementLevel` (I prefer the latter since it's clearer)

Reply:
changed to `AdditionToOuterLayerRadialRefinementLevel`